cmd/createSnapshot.go

```diff
@@ -80,12 +80,12 @@ func createSnapshotExec(cmd *cobra.Command, args []string) error {
 		return util.PxErrorMessage(err, "Failed to create snapshot")
 	}
 
-	// TODO: Show output in appropriate format
-
 	// Show user information
-	util.Printf("Snapshot of %s created with id %s\n",
+	msg := fmt.Sprintf("Snapshot of %s created with id %s\n",
 		csOpts.req.GetVolumeId(),
 		resp.GetSnapshotId())
+	output, _ := cmd.Flags().GetString("output")
+	util.PrintCreateOutput(output, "Create Snapshot", resp.GetSnapshotId(), msg)
```

Comment:
I think this is a good idea for commands that return only and Id, but instead of util.PrintCreateOutput taking many parameters we may have to adjust in the future, it should take a pointer to `util.FormatOutputId{}`

Reply:
I have done this as per my understanding of what  you requested for. Pleas review and let me know if this is what you wanted.